packages/react/src/components/TileGallery/__snapshots__/TileGallery.story.storyshot

```diff
@@ -13,7 +13,7 @@ exports[`Storybook Snapshot tests and console checks Storyshots 1 - Watson IoT/T
   >
     <div
       className="page-title-bar"
-      data-testid="page-title-bar"
```

Comment:
Not a breaking change. Just added in a previous testId PR. https://github.com/carbon-design-system/carbon-addons-iot-react/blame/c6d4413d8d07e9183d81faa41b606a33f903e299/packages/react/src/components/TileGallery/__snapshots__/TileGallery.story.storyshot